manifests/t/TermoraDev/Termora/1.0.5/TermoraDev.Termora.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: TermoraDev.Termora
+PackageVersion: 1.0.5
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{2E166884-7023-3703-8143-FD29FF19B328}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/TermoraDev/termora/releases/download/1.0.5/termora-1.0.5-windows-x86-64.msi
+  InstallerSha256: D41D3083A5D7C38111520D5646BFFBCFB2694A468BF6C82D3B9D3488BB75EB66

```

```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/TermoraDev/termora/releases/download/1.0.5/termora-1.0.5-windows-x86-64.msi
  InstallerSha256: D41D3083A5D7C38111520D5646BFFBCFB2694A468BF6C82D3B9D3488BB75EB66
- Architecture: x86
  InstallerUrl: https://github.com/TermoraDev/termora/releases/download/1.0.5/termora-1.0.5-windows-x86-64.msi
  InstallerSha256: D41D3083A5D7C38111520D5646BFFBCFB2694A468BF6C82D3B9D3488BB75EB66
```
Since both architectures are supported, both should be listed

---

I belive you include it after the URL
`<url>|x64 <url>|x86`